src/bank03_npc.asm

```diff
@@ -821,6 +821,7 @@ setNpcSpawnTable:
     pop  HL                                            ;; 03:4486 $e1
     ret                                                ;; 03:4487 $c9
 
+IF DEF(RNG_ORIGINAL)
```

Comment:
With the bank 00 changes moved to $1c44 this conditional should not be necessary, correct?

Reply:
Moved